packages/web3-eth-ens/src/registry.ts

```diff
@@ -48,23 +47,6 @@ export class Registry {
 		}
 	}
 
-	public setOwner(
```

Comment:
As these are breaking changes so if you have time this should be mentioned in same PR in **migration guide** ? or we can add in some other PR?